src/popover/src/Popover.js

```diff
@@ -168,7 +168,7 @@ export default class Popover extends Component {
 
   onBodyClick = e => {
     // Ignore clicks on the popover or button
-    if (this.targetRef === e.target) {
+    if (this.targetRef === e.target || this.targetRef.contains(e.target)) {
```

Comment:
`contains` will check the node itself as well, so you can make this:

```js
if (this.targetRef.containers(e.target)) {
```

For instance, this will be true: 
```js
const node = document.getElementById('foo')
node.contains(node)
```

That said, it doesn't read as explicit as you have it here, so I'm not opposed to this.

Reply:
Oh, you might also encounter a race condition where `targetRef` is actually undefined still, so it might be safe to check that it has been defined first:

```js
if (this.targetRef && this.targetRef.contains(e.target)) {
```